### ISSUE TITLE
DTLZ7

### ISSUE THREAD

**Issue by dongdafeng** [msu-coinlab/pymop#6]
Hi, DTLZ7 does not implement the '`_calc_pareto_front`' method.

**Comment by julesy89** [comment]
yes so far I added only for the problems with a linear or spherical pareto front.
I will have a look if I do have the mathematical expression for the front somewhere...
If you are planning to implement it by yourself, feel free to send it to me via email or directly create a pull request.

Julian
